cns/middlewares/k8sSwiftV2_linux.go

```diff
@@ -86,15 +50,57 @@ func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
 	return nil
 }
 
-func addRoutes(cidrs []string, gatewayIP string) []cns.Route {
-	routes := make([]cns.Route, len(cidrs))
-	for i, cidr := range cidrs {
-		routes[i] = cns.Route{
-			IPAddress:        cidr,
-			GatewayIPAddress: gatewayIP,
-		}
+// CNS gets pod CIDRs from configuration env and parse them to get the v4 and v6 IPs
+// Containerd reassigns the IP to the adapter and kernel configures the pod cidr route by default, so windows swiftv2 does not require pod cidr
+func (k *K8sSWIFTv2Middleware) GetPodCidrs() (v4IPs, v6IPs []string, err error) {
```

Comment:
I think you moved this into the linux impl from the previous pr but the code to get the pod cidrs themselves isn't linux-specific, right? Like: would this code still work on windows (even if we don't use it on windows)? If the above is yes, it can remain in the common golang file (and if so move that comment about windows on line 54 to getInfraRoutes or an os-specific function), otherwise you can ignore this